### PR TITLE
DROOLS-1526: adding rule and compilation check to report empty decision table input entries

### PR DESCRIPTION
**Pull request by etirelli**
@tarilabs can you please review

### CODE REVIEW

**Comment by etirelli** [comment]
jenkins test again

File: kie-dmn/kie-dmn-validation/src/test/resources/org/kie/dmn/validation/DTABLE_EMPTY_ENTRY.dmn [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      </input>
+      <output id="_oApprovalStatus">
+        <!-- TEST -->
+      </output>
+      <rule id="_7f03803d-2636-40ab-8346-7fd7f38ab695">
+        <inputEntry id="_7f03803d-2636-40ab-8346-7fd7f38ab695-0">
+          <text>&gt;=18</text>
+        </inputEntry>
+        <inputEntry id="_7f03803d-2636-40ab-8346-7fd7f38ab695-1">
+          <text/>
```
Comment (tarilabs) [inline kie-dmn/kie-dmn-validation/src/test/resources/org/kie/dmn/validation/DTABLE_EMPTY_ENTRY.dmn:60]: Assumed this is the deliberate issue under test.

**Comment by tarilabs** [comment]
Please consider closing this as #1211 replacement keeping the separate commits.

File: kie-dmn/kie-dmn-validation/src/test/resources/org/kie/dmn/validation/DTABLE_EMPTY_ENTRY.dmn (hunk 1 above)
Comment (etirelli) [reply kie-dmn/kie-dmn-validation/src/test/resources/org/kie/dmn/validation/DTABLE_EMPTY_ENTRY.dmn:60]: Yes, correct.

**Comment by etirelli** [comment]
closing as this was replaced by PR #1211

**Pull request closed without merging**